build.gradle

```diff
@@ -0,0 +1,12 @@
+plugins {
+    id 'java-library'
+}
+
+group 'org.htsjdk'
+version '0.0.1'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
```

Comment:
We can also add some basic dependencies already: concretely, I would propose to add [SLF4J](https://www.slf4j.org/) as a logging system (users can choose whatever logging they want, and how to set the log level) and maybe [Apache commons-compress](https://commons.apache.org/proper/commons-compress/) to support different compression for files out-of-the-box (we can also create a compressor in the library for bgzip and the gzip compressor from commons-compress might be a better solution to gzip due to the bugs in the `java.util.zip` implementation).

If this is too controversial, we can let the dependencies for another PR (I will open an issue for the logging system anyway for initiate discussion).

Reply:
Open #15 for slf4j